.github/pull_request_template.md

```diff
@@ -14,15 +13,12 @@ Link the issues to be closed with this PR, whether in this repository, or in ano
 - fixes #<issue_number>
 - fixes noaa-emc/fv3atm/issues/<issue_number>
 
-
-
 ## Testing
 
 How were these changes tested?
 What compilers / HPCs was it tested with?
 Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
-
-
+Have regression tests and unit tests (utests) been run? On which platforms and with which compilers?
```

Comment:
Utests can be run on Hera, Orion, Dell with Intel compilers (I had some trouble with ecFlow on Cray)

Reply:
Should I add this information to the PR template?

Reply:
Since regression tests are supported on all platforms (tier 1, 2, etc.), maybe it would be helpful to add that utests are only supported on tier 1 (Cray will have to come later)? e.g. unit tests (utests on tier 1 platform)